src/main/java/org/jenkinsci/test/acceptance/docker/Docker.java

```diff
@@ -65,6 +66,26 @@ public boolean isAvailable() {
     }
 
     /**
+     * Checks if a given container is currently running.
+     *
+     * @param container Container id
+     * @return false if the container is not running, true otherwise
+     */
+    public boolean isContainerRunning(String container) throws IOException, InterruptedException {
+        ProcessBuilder pBuilder = cmd("ps").add("-q", "--filter", "\"id=" + container + '"').build();
+        Process psProcess = pBuilder.start();
+        psProcess.waitFor();
+        if (psProcess.exitValue() == 0) {
```

Comment:
Well I did not know it but `waitFor` returns the exit status - so this can be simplified.